Makefile

```diff
@@ -32,21 +32,21 @@ build-vms:
 build-benchs: build-krun
 	cd benchmarks && \
 		env LD_LIBRARY_PATH=${GCC_LIB_DIR} \
-		${MAKE} CC=${CC} JAVAC=${JAVAC}
```

Comment:
Why are they `make` variables and not environment variables?

[The Makefile didn't even work, at least, on Linux before. I assume -- but don't know for sure -- that if you specify make variables you need to specify a target. And at least one of the commands specified `${MAKE}` twice, which was obviously wrong, on top of that.]